### PR TITLE
Bicaws 1878 publish npm package

### PR DESCRIPTION
**Pull request by C-gyorfi**
- Add GH actions workflow to release NPM package when pushing to main:
  - Package is built from the `output-data` folder

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - BICAWS-1878-publish-npm-package
```
Comment (daviesjamie) [inline .github/workflows/release.yml:6]: We should probably remove this branch from the config before merging to main?

File: output-data/index.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import amountData from "./data/amount.json"
+
+const actualOffenceDate: ActualOffenceDate[] = actualOffenceDateData
+const alcoholLevelMethod: AlcoholLevelMethod[] = alcoholLevelMethodData
+const amount: Amount[] = amountData
+
+export default {
+  actualOffenceDate,
+  alcoholLevelMethod,
+  amount
```
Comment (daviesjamie) [inline output-data/index.ts:14]: Why aren't we also importing/typing/exporting the other JSON files?

File: README.md [hunk 3]
```diff
@@ -13,3 +13,13 @@ The output offence code data is generated by combining a number of different dat
 ## Consistently Formatting Data
 In order to make differences between versions of data easy to read, the data should be sorted before committing. This can be done by running `./data-formatter/format.sh` from the root of the repository. This will sort the arrays of data by alphabetical attribute name and then output them with their attributes sorted.
 
+## Publishing
+The output-data lib is published as an NPM package to [GitHub packages](https://github.com/features/packages) using a [GitHub Actions](https://github.com/features/actions) workflow `.github/workflows/release.yml`. 
+
+Versioning and publishing is automated using [semantic-release](https://github.com/semantic-release/semantic-release). The following commit convention convention triggers releases:
```
Comment (daviesjamie) [inline README.md:19]: So are we saying that the commit message has to match a certain format in order for the npm package to be released?

Comment (C-gyorfi) [reply README.md:19]: Yes, more [info](https://github.com/semantic-release/semantic-release#Commit-message-format)

File: output-data/index.ts (hunk 2 above)
Comment (C-gyorfi) [reply output-data/index.ts:14]: Will do it the next slice

File: README.md (hunk 3 above)
Comment (daviesjamie) [reply README.md:19]: What happens if the commit message doesn't match the format? Do we release with a default release type? Or does nothing get released?

**Review by daviesjamie — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (C-gyorfi) [reply README.md:19]: Nothing gets released

**Pull request merged**